src/pages/transaction-signing/components/actions.tsx

```diff
@@ -84,35 +93,59 @@ const SubmitActionSuspense = (props: ButtonProps) => {
   }, [setIsLoading, setIsIdle, handleBroadcastTransaction]);
 
   return (
-    <BaseConfirmButton
-      data-testid={TransactionsSelectors.BtnConfirmTransaction}
-      onClick={handleSubmit}
-      isLoading={isLoading}
-      isDisabled={isDisabled}
-      {...props}
-    >
-      Confirm
-    </BaseConfirmButton>
+    <>
+      <BaseConfirmButton
+        data-testid={TransactionsSelectors.BtnConfirmTransaction}
+        onClick={handleSubmit}
+        isLoading={isLoading}
+        isDisabled={isDisabled}
+        {...props}
+      >
+        Confirm
+      </BaseConfirmButton>
+    </>
   );
 };
+
 const SubmitAction = (props: ButtonProps) => {
   return (
-    <React.Suspense fallback={<BaseConfirmButton isLoading isDisabled {...props} />}>
-      <SubmitActionSuspense {...props} />
-    </React.Suspense>
+    <>
+      <React.Suspense fallback={<BaseConfirmButton isLoading isDisabled {...props} />}>
+        <SubmitActionSuspense {...props} />
+      </React.Suspense>
+      <React.Suspense fallback={<ShowTxSettingsPlaceholder />}>
+        <ShowTxSettingsAction />
+      </React.Suspense>
+    </>
   );
 };
 
 const FeeRowItemSuspense = () => {
+  let showWarning = false;
+  const fee = useTransactionFee();
+  const transactionRequest = useTransactionRequest();
+  const appName = transactionRequest?.appDetails?.name;
+  const customFee = transactionRequest?.fee;
+
+  if (!!customFee && fee.amount) {
+    showWarning = customFee > fee.amount * 4;
```

Comment:
Currently, `customFee` and `fee.amount` here are the same so this will never show. Working to fix.